### ISSUE TITLE
Is this project still under development?

### ISSUE THREAD

**Issue by linusang** [evnbr/regionize#8]
Hi, I'm wondering if this project is under development? Is there a working demo I can play and understand how to use this library?

**Comment by evnbr** [comment]
Hi @linusang — this project is on a bit of hiatus, but I hope to return to it at some point. Right now its only use is the underlying engine for https://github.com/evnbr/bindery , would check that out for working demos.

The idea is that this library could works for anything region-like and bindery is very book/print-specific, but that's fairly untested.

Curious if you're looking for something specific, happy to answer any questions about if regionize is a good fit!

**Comment by linusang** [comment]
yeah i'm looking for a library to help split content between pages like google docs. i am looking to build a document editor that can paginate content. do you think regionize is a good fit?

**Comment by evnbr** [comment]
Sounds cool! Hm, not sure. This library just operates directly on static HTML, and does a single pass without listening for updates. So doing anything interactive— like seeing content reflow as you edit— would not be trivial. Perhaps it could work as a one-time mode like "print preview" but doesn't sound like thats what you had in mind.
